requirements.txt

```diff
@@ -1,31 +1,31 @@
-asdf==2.7.1
+asdf==2.8.1
 astropy==4.2.1
-astroquery==0.4.1
-authlib==0.15.3
+astroquery==0.4.2
+authlib==0.15.4
 bokeh==2.3.2
 codecov==2.1.11
-crds==10.3.11
+crds==11.0.4
 cryptography==3.4.7
-django==3.1.7
+django==3.2.5
```

Comment:
It looks like `django` is 3.2.4 in the python 3.8 yml file and 3.1.7 in the python 3.7 yml file. Should this be reverted to an earlier version in `requirements.txt`?